examples/analytics-vendors.amp.html

```diff
@@ -734,6 +779,12 @@
 <div class="logo"></div>
 <h1 id="top">AMP Analytics</h1>
 
+<p>
+<button id="fb-pixel">Track click with Facebook Pixel custom event</button> | <button id="fb-addToWishlist">Track Add To Wishlist</button>
```

Comment:
Please do not add another button for single analytics vendor testing. Use our `#test1` element to test click event instead.